src/components/markdown_editor/plugins/markdown_link_validator.test.tsx

```diff
@@ -72,4 +72,19 @@ describe('mutateLinkToText', () => {
       }
     `);
   });
+  it('keeps only the email address when handling mailto: links', () => {
```

Comment:
Worth clarifying this as well, since the email address is in 2 places in this test
```suggestion
  it('keeps only the link copy when handling mailto: links', () => {
```

Reply:
The duplicated email in this case is because the `link` node,

```
{
        type: 'link',
        url: 'mailto:someone@elastic.co',
        title: null,
        children: [{ value: 'someone@elastic.co' }],
}
```

is the representation of `[someone@elastic.co](mailto:someone@elastic.co)`, which is the format remark parses text email addresses into. As a bonus, this means both `someone@elastic.co` and `[someone@elastic.co](mailto:someone@elastic.co)` are handled by the same code path.